graveyard.json

```
@@ -1,4 +1,11 @@
 [
+  {
+    "dateClose": "2015-12-21",
+    "dateOpen": "2005-09-14",
+    "description": "Google Blog Search API was a way to search blogs utilizing Google.",
+    "link": "https://en.wikipedia.org/wiki/Google_Blog_Search",
+    "Google Blog Search API"
```

```suggestion
    "name": "Google Blog Search API"
```
Looks like the `name` key was missed here.

---

Oh whoops, didn't catch that, and nice ,you are using that new suggestion feature!

---

Thank you